pkgs/top-level/wine-packages.nix

```diff
@@ -58,11 +58,9 @@ rec {
   stagingFull = full.override { wineRelease = "staging"; };
 
   wayland = base.override {
-    wineRelease = "wayland";
-    waylandSupport = true;
+    x11Support = false;
   };
   waylandFull = full.override {
-    wineRelease = "wayland";
-    waylandSupport = true;
+    x11Support = false;
```

Comment:
When I started this PR I originally deprecated the wayland variants of wine, but since the wayland backend still isn't enabled by default, I decided to leave them in as X-less variants.